hello-1/test_package/conanfile.py

```diff
@@ -1,14 +1,16 @@
-from conans import ConanFile
+from conans import ConanFile, tools
 import os
 
 class PkgTestConan(ConanFile):
-    requires = "hello/0.1@armin/testing" % (username, channel)
+    requires = "hello/0.1@armin/testing"
     generators = "qmake"
+    settings = "os", "compiler", "arch", "build_type"
 
     def build(self):
-        os.chdir(self.conanfile_directory)
-        self.run('qmake')
-        self.run("make")
+        self.run('qmake ../.. -tp vc')
+        vcvars = tools.vcvars_command(self.settings)
+        self.run("%s && msbuild example.vcxproj /p:configuration=debug" % vcvars)
 
     def test(self):
-        self.run("%s%sexample" % (self.conanfile_directory, os.sep))
+        # #self.conanfile_directory
+        self.run(".%sdebug%sexample" % (os.sep, os.sep))
```

Comment:
The binary is in the temp directory, not in the conanfile_directory. The debug path is also hardcoded here.